docker/linux/unittest/Dockerfile

```diff
@@ -65,27 +65,27 @@ RUN cd /tmp \
     && rm clang-${CLANG_VER}.tgz
 
 # === Install Clang 16 toolchain for llvm-symbolization.
```

Comment:
Same nit as above, clang 16->17 now. Although maybe we should just say "Clang" or "Chrome/Cobalt Clang" or something more general.

Reply:
done